dhcpv4/dhcpv4.go

```diff
@@ -248,6 +248,17 @@ func NewRequestFromOffer(offer *DHCPv4, modifiers ...Modifier) (*DHCPv4, error)
 	)...)
 }
 
+// NewInformFromOffer builds a new DHCPv4 Informational message from the given
+// offer with the specified hardware address.
+func NewInformFromOffer(offer *DHCPv4, modifiers ...Modifier) (*DHCPv4, error) {
```

Comment:
With this we end up having `NewInformForInterface`, `NewInform` and `NewInformFromOffer`.
Do we really need all of them?
Do we really need to create an Inform based on an offer?

Reply:
You're right, we don't need all of them. The existing `NewInform` is sufficient, see the example in the updated description. I've removed `NewInformFromOffer`.